<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStud</class>
 <widget class="QWidget" name="AddStud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="photoLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="photoPathLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Photo path...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPhotoButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить фото</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Имя</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="birhtLabel">
      <property name="text">
       <string>Дата р.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="classLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="classLabel">
      <property name="text">
       <string>Группа</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="mailLabel">
      <property name="text">
       <string>Почта</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="telephoneLabel">
      <property name="text">
       <string>Телефон</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="mailLineEdit"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="telephoneLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
